_gsdocs-proposals/2020/proposal_AllpixSquaredRestructureDocs.md

```diff
@@ -0,0 +1,57 @@
+
+With over 3.5 years of continuous development and additions, the documentation has become a bit cluttered and we would like to streamline it with your help:
+
+* **Content:** We would like to separate more clearly which parts of the documentation are intended for users which are not expected to alter a single line of code - and which parts aim at developers who would like to contribute to the project or would like to extend their own copy. Also, we would like to review the current user manual for potentially lacking information or areas where we could improve.
+* **Infrastructure:** We would like to streamline the way we present the documentation and especially improve its online representation. Optimally we would end up with a streamlined, fully-integrated online presence which coherently presents news, manual and code reference. One possibility for this would e.g. be moving the documentation to [Docsy.dev](https://www.docsy.dev/). The challenging part about this is that we would like to keep as much as possible of our current automatic deployment and workflow.
+* **Bonus task:** For those of you who would like to dive a bit deeper into the scientific depth of this project - we currently do have a few examples for users to understand how to start using the framework and how to use certain features, but they are not very nicely presented and not very extensively documented. They could use a good polishing. Ultimately, these and further examples can be streamlined to follow a tutorial-like workflow.
+
+## Project duration
+
+We are open for both standard-length projects (3 months) and long-running (6 months) projects, depending on what you think is required to achieve the results we are aiming for.
+Given the good overall level of documentation of the project, we think a standard project might suffice.
+
+## Related material
+
+  * Project website: [http://cern.ch/allpix-squared](http://cern.ch/allpix-squared)
+  * Repository of website (hugo): [APSQ Website @ CERn GitLab](https://gitlab.cern.ch/allpix-squared/allpix-squared-website)
+  * Project repository: [Allpix Squared @ CERN GitLab](https://gitlab.cern.ch/allpix-squared/allpix-squared)
+  * Documentation within the repository: [doc/usermanual](https://gitlab.cern.ch/allpix-squared/allpix-squared/-/tree/master/doc%2Fusermanual)
+  * Online documentation: [Online User Manual](https://project-allpix-squared.web.cern.ch/project-allpix-squared/usermanual/allpix-manual.html)
+  * PDF documentation: [PDF User Manual](https://project-allpix-squared.web.cern.ch/project-allpix-squared/usermanual/allpix-manual.pdf)
+  * Doxygen code reference: [Code Reference](https://project-allpix-squared.web.cern.ch/project-allpix-squared/reference)
+
+## Expected results
+
+* A restructured user manual which clearly separates content intended for users and developers
```

Comment:
Why not a "User manual" and a "Developer guide", complementary to the Reference guide built with Doxygen? There are other projects having documentation split like this, e.g Geant4.

Reply:
We have discussed this but weren't entirely sure - the project isn't as big as Geant4 or others, and the development happens almost exclusively in close collaboration. Also, there is not so much information that a separate document is justified I guess.

Maybe something we can discuss with the candidate? I rephrased it accordingly.